### PR TITLE
PXC-3133 mysqld.cnf: remove log_error_verbosity option

### PR DESCRIPTION
**Pull request by vorsel**
In order not to overload logs, remove log_error_verbosity=3 from mysqld.cnf file and use
default value of 2.

### CODE REVIEW

**Review by EvgeniyPatlan — APPROVED** [review]
:+1:  but please wait for @vadimtk approval

**Review by vadimtk — APPROVED** [review]

**Pull request merged**
